config/initializers/delayed_job.rb

```diff
@@ -3,4 +3,4 @@
 Delayed::Worker.max_run_time = 5.minutes
 Delayed::Worker.logger = Logger.new(Rails.root.join('log/delayed_job.log'))
 
-PRIORITIES = { push: 1, download: 2, web_hook: 3 }.freeze
+PRIORITIES = { push: 1, download: 2, web_hook: 3, download_metrics: 4 }.freeze
```

Comment:
I hate this constant been here :( eventually i wanna moved it out. I had that somewhere. Anyways, not in the scope of this PR